discover/portal/articles/125-workflow/00-intro.markdown

```diff
@@ -31,8 +31,27 @@ After all that, you should be familiar with how to use Liferay's Kaleo workflow
 to set up approval process for any kind of content before it is published to
 your portal. 
 
+## Embedded Workflows [](id=embedded-workflows)
+
+In addition to the Single Approver definition, there are some workflow
+definitions that ship with Liferay but are not pre-installed, since they're
+primarily included for test cases. They can be found in the Liferay source code
+in
+`modules/aps/forms-and-workflow/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/resources/META-INF/definitions`,
+or in your Liferay installation itself. Open your Liferay installation's
+`osgi/marketplace/Liferay CE Forms and Workflow.lpkg`, and then find and open
+the `com.liferay.workflow.kaleo.runtime.impl-[version].jar`. The definitions are
+in the `META-INF/definitions` folder (see the diagrams
+[here](/develop/tutorials/-/knowledge_base/6-2/designing-a-kaleo-workflow-definition)).
+Once you have the XML definition files locally, you can upload them into
+Liferay. Check out the article on uploading definitions to do that <!--LINK-->.
```

Comment:
Were you going to provide this link?

Reply:
Better to just describe here. This never really needed its own article.
Added this commit: https://github.com/sez11a/liferay-docs/pull/3040/commits/a534591ca08f100f55c84327c58885c29e86d09d